backend/src/models/User.ts

```diff
@@ -1,25 +1,26 @@
-import mongoose, { Schema } from "mongoose";
+import mongoose, { Schema } from 'mongoose';
 
-export interface IUser {
-  fullName: string;
-  email: string;
-  photoUrl: string;
-  roles: string;
-}
+export interface IUser extends Document {
+    _id: Schema.Types.ObjectId;
+    fullName: string;
+    email: string;
+    photoUrl: string;
+    roles: string;
```

Comment:
```suggestion
    role: string;
```
This should be role since `UserSchema` use `role` not `role`, which leads to IDE wrongly suggest on `TestController`